tools/measurement/apksize/apksize.gradle

```diff
@@ -42,6 +42,15 @@ task generateApkSizeMeasurements(type: GenerateMeasurementsTask) {
     reportFile = file("$buildDir/size-report.json")
 }
 
+task generateAarSizeMeasurements(type: com.google.firebase.gradle.plugins.measurement.aarsize.GenerateMeasurementsTask) {
```

Comment:
I'm not an expert of groovy. Does it support import aliases like python or typescript?

Reply:
I didn't know either, but it does. Fixed!